mtags/src/main/scala/scala/meta/internal/mtags/OnDemandSymbolIndex.scala

```diff
@@ -11,6 +11,7 @@ import scala.meta.internal.semanticdb.Scala._
 import scala.meta.internal.{semanticdb => s}
 import scala.meta.io.AbsolutePath
 import scala.util.control.NonFatal
+import scala.meta.internal.mtags.ListFiles
```

Comment:
I'm not sure what was happening here, but it seemed that this `ListFiles` was being shadowed by `scala.meta.internal.io.ListFiles`